packages/story-editor/src/components/form/tags/input.js

```diff
@@ -161,14 +220,39 @@ function Input({
           )
         )
       }
+      {suggestedTerms.length > 0 && value.length >= 3 && (
```

Comment:
you should be able to remove the `value.length >= 3` since that's an implementation detail covered in when we fetch results vs how we display results